pkg/kubelet/dockertools/manager.go

```diff
@@ -1156,6 +1156,7 @@ func (dm *DockerManager) killContainer(containerID types.UID) error {
 	if inspect != nil && inspect.Config != nil && inspect.Config.Labels != nil {
 		preStop, found = inspect.Config.Labels[kubernetesPodLabel]
 	}
+	gracePeriod := uint(30)
```

Comment:
Accidental, will move it there.

> On May 21, 2015, at 7:15 PM, Brian Grant <notifications@github.com> wrote:
> 
> In pkg/kubelet/dockertools/manager.go:
> 
> > @@ -1156,6 +1156,7 @@ func (dm *DockerManager) killContainer(containerID types.UID) error {
> >  	if inspect != nil && inspect.Config != nil && inspect.Config.Labels != nil {
> >  		preStop, found = inspect.Config.Labels[kubernetesPodLabel]
> >  	}
> > +	gracePeriod := uint(30)
> Why is this defaulting in kubelet rather than in the API defaulting code? I'd prefer the latter.
> 
> —
> Reply to this email directly or view it on GitHub.
>